packages/@triplex/server/src/ast/__tests__/type-infer.test.ts

```diff
@@ -1201,4 +1201,37 @@ describe("type infer", () => {
         ]
       `);
   });
+  it("should evaluate expressions", () => {
+    const project = _createProject({
+      tsConfigFilePath: join(__dirname, "__mocks__/tsconfig.json"),
+    });
+    const sourceFile = project.addSourceFileAtPath(
```

Comment:
nit: you can use `createSourceFile` and have it embedded in the test instead of having it in the mocks folder

tbh I would have done that but didn't realize it existed until much later